src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+function moveFile(from, to) {
+  const sourcePath = path.resolve(from);
+  const destinationPath = path.resolve(to);
+
+  if (!fs.existsSync(sourcePath)) {
+    console.log('Source file does not exist.');
+
+    return;
+  }
+
+  const directory = path.dirname(destinationPath);
+
+  if (!fs.existsSync(directory)) {
+    console.log('Destination directory does not exist.');
+
+    return;
+  }
+
+  const finalDestinationPath = destinationPath.endsWith('/')
```

Comment:
In the new path for the file `/` will not necessarily be at the end (see the example in the task), so you need to check if it is there at all.